### DISCUSSION TITLE
User endpoints are ready 🎉

### DISCUSSION THREAD

**Discussion by Meminseeker** [bounswe/bounswe2024group11#200] [Announcements]
With @ceydanursen, we wrote the endpoints for the User model. All the CRUD operations are working.

Please note that more tests are needed to say it is finished, and we have not yet merged it with the development branch.

@ArdaVural @ceydanursen @tenacke @hks1444 @Meminseeker @meminciftci @MucahitErdoganUnlu @ozankrkya @yunuskaydin @umitcan07

**Comment by Meminseeker** [comment]
For those who need to see, validate, or use mock data for the User model, you can use this:

```JSON
{
        "id": 1,
        "username": "username1",
        "email": "user1@email.com",
        "name": "updated name1",
        "surname": "updated surname1",
        "created_at": "2024-04-20T18:44:32.634562Z"
}
```

**Comment by umitcan07** [comment]
Lovely!
